.github/workflows/test.yml

```diff
@@ -34,6 +34,10 @@ jobs:
       - name: Install libraries with dev group
         run: |
           poetry install --with dev
+      - name: Install with pipx
+        run: |
+          pipx ensurepath
+          pipx install git+https://github.com/Pennyw0rth/NetExec --python python${{ matrix.python-version }}
```

Comment:
According to https://github.com/actions/checkout#checkout-v4 the netexec repository should be located under `${{ github.workspace }}`. Let's use that for the installation instead of having to download it again.